src/app/components/select-field/index.html

```diff
@@ -88,6 +95,12 @@
   </sky-demo-page-properties>
 
   <sky-demo-page-properties sectionHeading="Select field events">
+    <sky-demo-page-property
+      propertyName="addNewRecordButtonClick"
+      isOptional="true"
+    >
+      Fires when users add new items directly from the select field modal.
```

Comment:
This event doesn't fire when new items are created (the consumer must implement that logic themselves), it only fires when the New button is clicked. Is it worth pointing out that distinction?

Reply:
I think it is. That was my best guess and was hoping to be corrected or validated during review. 😆